spinalcordtoolbox/deepseg/models.py

```diff
@@ -249,8 +259,25 @@
                              'tandem with another model for specialized cord localisation of spinal cords with tumors '
                              '(https://github.com/ivadomed/findcord_tumor).',
          'url': 'https://github.com/ivadomed/model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel',
-         'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel']},
-    'seg_exvivo_gm-wm_t2':
+         'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel'],
+         'citation': textwrap.dedent("""
+             ```
+             @article{LEMAY2021102766,
+                      title={Automatic multiclass intramedullary spinal cord tumor segmentation on MRI with deep learning},
+                      journal={NeuroImage: Clinical},
+                      volume={31},
+                      pages={102766},
+                      year-2021},
+                      issn-2213-1582},
+                      doi-https://doi.org/10.1016/j.nicl.2021.102766},
+                      url-https://www.sciencedirect.com/science/article/pii/S2213158221002102},
+                      author-Andreanne Lemay and Charley Gros and Zhizheng Zhuo and Jie Zhang and Yunyun Duan and Julien Cohen-Adad and Yaou Liu},
+                      keywords-Deep learning, Automatic segmentation, Spinal cord tumor, MRI, Multiclass, CNN}
```

Comment:
Haha, trying out syntax highlighting on this snippet made me notice some syntax errors:
```suggestion
                      year={2021},
                      issn={2213-1582},
                      doi={https://doi.org/10.1016/j.nicl.2021.102766},
                      url={https://www.sciencedirect.com/science/article/pii/S2213158221002102},
                      author={Andreanne Lemay and Charley Gros and Zhizheng Zhuo and Jie Zhang and Yunyun Duan and Julien Cohen-Adad and Yaou Liu},
                      keywords={Deep learning, Automatic segmentation, Spinal cord tumor, MRI, Multiclass, CNN}
```